pkgs/development/python-modules/pypdfium2/default.nix

```diff
@@ -3,21 +3,22 @@
   pkgs,
   buildPythonPackage,
   fetchFromGitHub,
-  fetchurl,
+  fetchgit,
   setuptools-scm,
   pdfium-binaries,
   numpy,
   pillow,
   pytestCheckHook,
-  python,
 }:
 
 let
   pdfiumVersion = "${pdfium-binaries.version}";
 
-  headers = fetchurl {
-    url = "https://pdfium.googlesource.com/pdfium/+archive/refs/heads/chromium/${pdfiumVersion}/public.tar.gz";
-    hash = "sha256-vKfs4Jd8LEtA3aTI+DcJMS0VOErq1IR1eThnMlxiER0=";
+  headers = fetchgit {
+    url = "https://pdfium.googlesource.com/pdfium";
+    # The latest revision on the chromium/${pdfiumVersion} branch
+    rev = "f6da7d235728aeaff6586d2190badfb4290a9979";
```

Comment:
How do we make sure this is update in tandem with pdfium-binaries?

Reply:
This I don't actually know. Do you have any ideas?

Reply:
Would an update script be reasonable? I can add one in a subsequent PR.

Reply:
> Do you have any ideas?

We should add a comment to the pdfium-binaries package reminding people to update this revision.

Reply:
I've added passthru for `pdfium-binaries`.

I also looked at adding `updateScript` but got stuck. The first hurdle was that `gitUpdater` deals with tags only, and here we'd need a branch name (then transformed into a revision). This I did, but then it wasn't clear to me how to make the script for `pypdfium2` update the `headers.rev` field. Perhaps @sarahec knows better how to bend the tool, so I'll leave this out of scope here for now.